processing/bloom_handler_test.go

```diff
@@ -46,6 +46,9 @@ func makeBloomDNSEvent(rrname string) types.Entry {
 		DNSType:   []string{"answer", "query"}[rand.Intn(2)],
 	}
 	eve := types.EveEvent{
+		Timestamp: &types.SuriTime{
+			Time: time.Now(),
```

Comment:
Not `time.UTCNow()`?

Reply:
This is code that just generates mock test input that would otherwise come from Suricata. The code being tested does not make use of the timestamp at all. It is only generated to have _some_ value in there to get a more realistic entry example. 
That being said, there's nothing to say against forcing UTC here. But I assume you meant ` time.Now().UTC()`, there is no `time.UTCNow()`?